src/main/java/seedu/ui/TextUi.java

```diff
@@ -121,4 +122,27 @@ public static void printLocalSearchMessage() {
         System.out.println(LINE + "!WARNING! This is a local search, data might not be up to date.");
         System.out.println(LINE);
     }
+
+    public static void printAddMessage(String moduleCode) {
+        System.out.println("Adding " + moduleCode);
+    }
+
+    public static void printLectureMessage() {
+        System.out.println("Which Lecture time slot would you like to add ?");
+        System.out.println("Lecture Time slots:");
+    }
+
+    public static void printTutorialMessage() {
+        System.out.println("Which Tutorial time slot would you like to add ?");
+        System.out.println("Tutorial Time slots:");
+    }
+
+    public static void printLaboratoryMessage() {
+        System.out.println("Which Laboratory time slot would you like to add ?");
+        System.out.println("Laboratory Time slots:");
+    }
+
+    public static void printLessonInfo(int serial, Lesson lesson) {
+        System.out.println(serial + ": " + lesson.lessonDetail());
+    }
```

Comment:
Referring to the 3 json files below and my previous comment, would it be better to move the jsons into /data/ so git does not track them for now?